concent_api/core/views.py

```diff
@@ -623,6 +624,13 @@ def deserialize_message(raw_message_data):
         raise Http400("Unable to deserialize Golem Message: {}.".format(exception))
 
 
+def validate_report_computed_task_time_window(report_computed_task):
+    assert isinstance(report_computed_task, message.ReportComputedTask)
+
+    if report_computed_task.task_to_compute.timestamp > report_computed_task.timestamp:
+        raise Http400("ReportComputedTask timestamp is older then nested TaskToCompute.")
```

Comment:
Please reverse this condition for readability:
``` python
report_computed_task.timestamp < report_computed_task.task_to_compute.timestamp
```

It's easier to parse mentally when it's the same direction in the condition and in the message. For a moment I even thought that there was an error here and wanted to report it.